src/main/java/com/artipie/Settings.java

```diff
@@ -80,13 +92,18 @@ public Fake(final Storage storage) {
         }
 
         @Override
-        public Storage storage() throws IOException {
```

Comment:
How those changes are related to the Pull Request?

Reply:
@Sammers21 it's just a minor cosmetic change in related class

Reply:
@g4s8, move this change to a separated Pull Request with `minor changes` since this change has no intersections with this Pull Request goals.